install_config/certificate_customization.adoc

```diff
@@ -16,9 +16,43 @@ Administrators can configure custom serving certificates for the public host
 names of the {product-title} API and
 xref:../architecture/infrastructure_components/web_console.adoc#architecture-infrastructure-components-web-console[web console].
 This can be done during an
-xref:../install_config/install/advanced_install.adoc#advanced-install-custom-certificates[advanced
-installation] or configured after installation.
+xref:../install_config/install/advanced_install.adoc#advanced-install-custom-certificates[advanced installation] or configured after installation.
 
+[[ansible-configuring-custom-certificates]]
```

Comment:
Playing connect the docs for other work https://github.com/openshift/openshift-ansible/pull/1142 is related, to Ansible and certificates and likely needs a docs location. Would another section on this doc be appropriate?